### ISSUE TITLE
Lack of Documentation

### ISSUE THREAD

**Issue by srand2** [heydc7/Tinker#1]
Hey. Would it be possible to add a How To Use section? It would contain how to install and then a couple of use cases. Thanks

**Comment by aditya936** [comment]
yes it would help a lot if you add the installation and usage instructions. Thnx in advance

**Comment by heydc7** [comment]
Hey, sorry for that. I will add documentation as soon as possible. Thank you for reaching out!
